src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheck.java

```diff
@@ -54,7 +54,7 @@
  * Property {@code tokens} - tokens to check
  * Type is {@code java.lang.String[]}.
  * Validation type is {@code tokenSet}.
- * Default value is: empty.
+ * Default value is: {@code ""}.
```

Comment:
if this is changed, how come metadata file didn't change?

Reply:
because it was in right form (with "" already)
https://github.com/checkstyle/checkstyle/blob/58bae0f3defcf7ba4a9fa69bed97aadcb35eb2ae/src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/IllegalTokenTextCheck.xml#L25-L28